pkg/volume/util/operationexecutor/operation_generator.go

```diff
@@ -347,7 +369,7 @@ func (og *operationGenerator) GenerateAttachVolumeFunc(
 
```

Comment:
This code needs to be updated. `MarkVolumeAsUncertain(...)` was added since this PR was opened.

Reply:
good catch, thanks